resources/js/Layouts/AppLayout.vue

```diff
@@ -344,6 +351,13 @@
                         >
                             Gruppe
                         </jet-responsive-nav-link>
+
+                        <jet-responsive-nav-link
+                            :href="route('app.post.index')"
+                            :active="'app.post.index' == currentRoute"
+                        >
```

Comment:
We can add a `v-if="$page.props.orga"` for now, no?

Reply:
yes, thanks